apps/projects/templates/meinberlin_projects/project_list.html

```diff
@@ -8,12 +8,10 @@
         {% include "meinberlin_contrib/includes/filter_and_sort.html" with filter=view.filter %}
 
         <div class="l-tiles-4">
-            {% for project in project_list %}
+            {% filter_has_perm 'a4projects.view_project' request.user project_list as filtered_projects  %}
```

Comment:
I wonder whether a syntax like `{% for project in project_list|has_perm:'a4projects.view_project' %}` would be nicer.